File: 4th Semester/Design and programming environments/Proiect C-Sharp Client-Server Sockets/AppModel/Inscriere.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppModel
{
    public class Inscriere : Entity<int>
    {
        public Inscriere(Participant a, Proba b)
        {
            Participant = a;
            Proba = b;
        }
        public Participant Participant { get; set; }

        public Proba Proba { get; set; }

        public override string ToString()
        {
            return Participant + " " + Proba;
        }
    }
}

File: 4th Semester/Design and programming environments/Project Java Client-Server/Model/src/main/java/domain/Inscriere.java
package domain;

public class Inscriere extends Entity<Integer>{
    private Participant participant;
    private Proba proba;

    public Inscriere(){

    }

    public Inscriere(Participant participant, Proba proba) {
        this.participant = participant;
        this.proba = proba;
    }

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public Proba getProba() {
        return proba;
    }

    public void setProba(Proba proba) {
        this.proba = proba;
    }
}
